KarhooUISDKTests/TestCases/Screens/GuestCheckoutScreen/KarhooGuestCheckoutPresenterSpec.swift

```diff
@@ -19,6 +19,7 @@ class KarhooGuestCheckoutPresenterSpec: XCTestCase {
     private var mockTripService = MockTripService()
```

Comment:
@mohadian wheren't there 2 checkout components before? One for guest and one for karhoo user / token exchange, each with its own `Spec`? If so, this entire file may be obsolete. Or at least we should cherry pick whatever tests are still valuable and add them to the `KarhooCheckoutPresenterSpec` before deleting this.